tests/test_backend.py

```diff
@@ -461,23 +461,31 @@ def test_ls(tmpdir, backend):
     assert backend.ls() == []
     assert backend.ls('/') == []
 
-    sub_content = [  # files in sub directory
-        ('sub/file.foo', '1.0.0'),
-        ('sub/file.foo', '2.0.0'),
+    content = [
+        ('file.bar', '1.0.0'),
+        ('file.bar', '2.0.0'),
+        ('file.foo', '1.0.0'),
     ]
-    sub_content_latest = sub_content[-1:]
-
-    content = [  # files in root directory
+    content_bar = [
         ('file.bar', '1.0.0'),
         ('file.bar', '2.0.0'),
     ]
-    content_latest = content[-1:] + sub_content_latest
-    content += sub_content
+    content_latest = [
+        ('file.bar', '2.0.0'),
+        ('file.foo', '1.0.0'),
+    ]
+    sub_content = [
+        ('sub/.file.foo', '1.0.0'),
+        ('sub/.file.foo', '2.0.0'),
```

Comment:
Could you also add a test for a hidden folder as different functions might be used in the backend to discover them (as it is the case in the current file-system backend).

Reply:
Good idea. I extended the tests accordingly.